xAPI.md

```diff
@@ -916,11 +914,11 @@ The details of each property of a statement are described in the table below.
 	<td>Context that gives the Statement more meaning. Examples: a team the Actor is 
 	working with, altitude at which a scenario was attempted in a flight simulator.</td>
 	<td>Optional</td></tr>
-	<tr><td><a href="#timestamp">timestamp</a></td><td>Date/Time</td>
+	<tr><td><a href="#timestamp">timestamp</a></td><td>Timestamp</td>
 	<td>Timestamp (Formatted according to <a href="https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations">ISO 8601</a>) 
 	of when the events described within this Statement occurred. Set by the LRS if not provided.</td>
```

Comment:
I'd like to see the contents of this cell relating to ISO 8601 and "set by the LRS" (and equivalent cell for stored) moved to the data type section and a link to that section round the word "Timestamp" in the type column. The description column here would then become simply "timestamp of when the events described within this statement occurred." and "Timestamp of when this statement was recorded".